gflownet/envs/ctorus.py

```diff
@@ -143,54 +173,106 @@ def get_parents(
             parents = [state]
             return parents, [action]
 
-    def sample_actions(
+    def action2representative(self, action: Tuple) -> Tuple:
```

Comment:
if this method returns  `self.representative_action` regardless the `action`, why is it `action2representative` with an argument `action`, not just `get_representative_action` (w/o any arguments)?